torch/csrc/lazy/ts_backend/ops/device_data.h

```diff
@@ -15,14 +15,29 @@ class TORCH_API DeviceData : public TsNode {
 
   explicit DeviceData(std::shared_ptr<BackendData> data);
 
+  // A DeviceData node can be reused if the shape matches,
+  // but we will substitute the actual data_ pointer under
+  // the hood.
+  bool CanBeReused(std::shared_ptr<BackendData> data) const {
```

Comment:
qq, seems like we need to add this method to all ops otherwise, do you plan to add it for all manual ops like view ops in this pr? What's going to happen if one op does not have `CanBeReused` defined and we try to reuse it?

Reply:
Yes, I will have another PR to add other manual ops. If we call ReuseNode on an unsupported op, the compilation would fail. The idea is that unsupported ops should not be inserted into TrieCache and thus IR resuing just ignores those ops.